jerry-main/main-unix-snapshot.c

```diff
@@ -230,17 +243,19 @@ process_generate (cli_state_t *cli_state_p, /**< cli state */
         snapshot_flags |= JERRY_SNAPSHOT_SAVE_STATIC;
         break;
       }
-      case OPT_GENERATE_LITERAL_LIST:
       case OPT_GENERATE_LITERAL_C:
+      case OPT_GENERATE_LITERAL_LIST:
+      case OPT_IMPORT_LITERAL_LIST:
       {
-        if (save_literals_file_name_p != NULL)
+        if (literals_file_name_p != NULL)
```

Comment:
Cannot we both load and save literals?

Reply:
Why? The loaded and saved list can be different. Although loading probably used only with static snapshots.

Reply:
I can change this, but it is not a valid use case. It will never happen to use them in the same command.